ccmlib/utils/version.py

```diff
@@ -81,6 +81,15 @@ def parse(version_string: str) -> re.Match:
             f"Cannot parse provided '{version_string}' scylla_version for the comparison. "
             f"Transformed scylla_version: {_scylla_version}")
 
+    def major(self) -> 'ComparableScyllaVersion':
```

Comment:
I'm curious why the return type is quoted?

Reply:
It can be removed, if using `from __future__ import annotations`

Which was supposed to the default in python 3.10, but it's not yet the default in any release

See for more details:
https://stackoverflow.com/questions/66734640/any-downsides-to-using-from-future-import-annotations-everywhere